src/lib/snykGroup.ts

```diff
@@ -69,6 +80,29 @@ export class snykGroup {
     } catch (err: any) {
       utils.log(err);
     }
+
+    //get roles
+    try{
+      let response = await this._requestManager.request({
+        verb: 'GET',
+        url:`/group/${this.id}/roles`
+      });
+      //map roles to role id
+      response.data.map( (currRole: any) => this._roles[currRole["name"].toUpperCase()] = currRole["publicId"]);
+      //if custom admin/collaborator role does not exist then translate admin/collaborator entry in membership file into that
```

Comment:
move this logic to a function, can use the map trick here as needed per the prior comment